res/CircleSchema/0.10/circle_schema.fbs

```diff
@@ -0,0 +1,1755 @@
+// Copyright (c) 2019~2025 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
This file is copied from https://github.com/Samsung/ONE/blob/9adb7178024fdba936c0887f3d8f923dd90ad48e/nnpackage/schema/circle_schema.fbs
+ Updated license year